__tests__/app.test.js

```diff
@@ -53,18 +53,17 @@ describe("GET /api", () => {
 
 })
 
-describe("GET /api/articles/:article_id", () => { 
+describe.only("GET /api/articles/:article_id", () => { 
     test("status:200, responds with article with a given id", () => {
         return request(app)
         .get("/api/articles/1")
         .expect(200)
         .then( ( { body: { article } }) => {
             testArticle = data.articleData[0]
             testArticle.created_at = "2020-07-09T20:11:00.000Z"
-            expect(article).toEqual( { 
-                                    article_id: 1,
-                                    ...testArticle
-                                })
+            testArticle.comment_count = "11"
```

Comment:
We shouldn't go back and amend previous tests when adding this new comment_count property. This is what is known as a brittle test. Rather than expecting our response to look exactly like an object, we can assert that it contains at least the properties we are interested in. Investigate toMatchObject and objectContaining jest matchers